README.md

```diff
@@ -47,6 +47,12 @@ node            |  |                               |  |
 celestia-appd --help
 ```
 
+### Environment variables
+
+| Variable        | Explanation                        | Default value | Required |
+| --------------- | ---------------------------------- | ------------- | -------- |
+| `CELESITA_HOME` | Home directory for the application |               | Optional |
```

Comment:
We should include the default value. I think we can just reference `os.UserHomeDir()` and link to the docs https://pkg.go.dev/os#UserHomeDir